sdk/monitor/monitor-opentelemetry/src/metrics/handler.ts

```diff
@@ -12,7 +12,6 @@ import {
 import { AzureMonitorOpenTelemetryConfig } from "../shared/config";
 import { _PerformanceCounterMetrics } from "./performanceCounters";
 import { _StandardMetrics } from "./standardMetrics";
-import { _NativeMetrics } from "./nativeMetrics";
```

Comment:
Why do these have underscores?

Reply:
Yeah I though that was a thing here in this repo for internal classes for some reason, it doesn't look that is the case so removing the underscore